src/silverlabnwb/silverlab.ophys.yaml

```diff
@@ -14,19 +14,18 @@ groups:
     dtype: int
     shape:
     - 2
-    doc: ' the 2d imaging frame size in voxels'
+    doc: the 2d imaging frame size in voxels
```

Comment:
Unrelated to these changes, but is this really voxels if we're imaging in 2d?

Reply:
Yea, it's debatable, I think... I could be wrong, but you could argue that since we are looking at a 3D image stack overall the data points we are looking at are voxels, and each imaging frame has, therefore, `frame_size[0]` voxels in one direction, and `frame_size[1]` voxels in the other. Guess it depends whether you want to emphasize the fact that the data points are 3d or that the frame is 2d more strongly?